apps/src/templates/progress/progressTestHelpers.js

```diff
@@ -0,0 +1,26 @@
+import _ from 'lodash';
+import { LevelStatus } from '@cdo/apps/util/sharedConstants';
+import { createStore } from 'redux';
+import Immutable from 'immutable';
+
+export const fakeLesson = (name, id, lockable=false) => ({name, id, lockable});
```

Comment:
Actually, a file-level comment to this effect would be helpful to keep people from trying to move this out of /src in the future.